example/stories/2-Card.stories.js

```diff
@@ -16,6 +16,7 @@ const Card = styled.div({
 export default {
   title: 'Card',
   component: Card,
+  parameters: { paddings: { disable: true } },
```

Comment:
You can remove this, I'll follow up with a PR refactoring example stories with different use cases.